src/common/components/CheckboxAggregation.jsx

```diff
@@ -1,16 +1,20 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import Immutable from 'immutable';
-import { List, Tag } from 'antd';
+import { List, Tag, Button } from 'antd';
 
 import CheckboxItem from './CheckboxItem';
 
+const BUCKET_CHUNK_SIZE = 10;
```

Comment:
It'd be better if we read this parameters from ``env``, it'll be much easier to configure them from OpenShift

Reply:
[create-react-app provides a way of doing this](https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/template/README.md#adding-custom-environment-variables)

But we need to create production build on the server, is that the case right now?

Reply:
can we just do for now smth like const

```js
BUCKET_CHUNK_SIZE = rocess.env.REACT_APP_SECRET_CODE || 20;
```